pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: func-deploy
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: CLI
+    tekton.dev/tags: cli
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    This Task performs deploy operation using Knative `func` CLI.
+  params:
+  - name: path
+    description: Path where the function project lives.
+    default: ""
+  workspaces:
+    - name: source
+      description: The workspace containing the `func` project.
```

Should probably be consistent with above (or change above to use `func`).
```suggestion
      description: The workspace containing the function project
```